README.md

```diff
@@ -99,16 +100,34 @@ Available commands:
   version  Show the version information.
 ```
 
-You can start a server by providing a path which contains multiple git repositories `/path/to/repositories` with this command:
+You can start a server providing a path which contains multiple git repositories `/path/to/repositories` with this command:
 
 ```
-$ gitbase server -v -g /path/to/repositories
+$ gitbase server -v -g /path/to/repositories -u gitbase
 ```
 
+### Docker
+
+You can use the official image from [docker hub](https://hub.docker.com/r/srcd/gitbase/tags/) to quickly run gitbase:
+```
+docker run --rm --name gitbase -p 3306:3306 -v /my/git/repos:/opt/repos srcd/gitbase:latest
+```
+
+If you want to speedup gitbase using indexes, you must run a pilosa container:
+```
+docker run -it --rm --name pilosa -p 10101:10101 pilosa/pilosa:v0.10.1
+```
+
+Then link the gitbase container to the pilosa one:
+```
+docker run --rm --name gitbase -p 3306:3306 --link pilosa:pilosa -e PILOSA_ENDPOINT="http://pilosa:10101" -v /my/git/repos:/opt/repos srcd/gitbase:latest
+```
+
+### Client
 A MySQL client is needed to connect to the server. For example:
 
 ```bash
-$ mysql -q -u root -h 127.0.0.1
+$ mysql -q -u root -h 127.0.0.1 -u gitbase
```

Comment:
Why changing the user's name? Besides now `-u` is set twice.

Reply:
The default user is changed in the Dockerfile to `gitbase`. I'm changing this to reflect what user you should use depending on gitbase running in local or docker.